src/quilla/common/enums.py

```diff
@@ -100,3 +100,13 @@ class OutputSources(Enum):
     LITERAL = 'Literal'
     XPATH_TEXT = 'XPathText'
     XPATH_PROPERTY = 'XPathProperty'
+
+
+class VisualParityImageType(Enum):
+    '''
+    The types of images that are supported by the
+    VisualParity validation state
+    '''
+
+    BASELINE = 'Baseline'
```

Comment:
This will be passed in to the plugin so that, if any plugin stores images differently based on the type of image, they know what type of image was passed

Reply:
We still need to store treatment image  - do we need a type for it?

Reply:
there is a type for treatment a well as baseline